Support building JVM / common without Android SDK

### Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Expected Behavior

Building the project on a system without Android SDK works.

### Current Behavior

The code in `buildSrc/android-sdk-detector.settings.gradle.kts` checks for the Android SDK and warns that it's not available. But building later fails when trying to configure kover.

### Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

#### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. clone repository an a machine without Android SDK
2. `./gradlew build`

#### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* MockK version: current main branch (1.13.8, `f879502a044c83c2a5fd52992f20903209eb34f3`)
* OS: Linux
* Kotlin version: -
* JDK version: 17
* JUnit version: -
* Type of test: -

#### Failure Logs

```
FAILURE: Build failed with an exception.

* Where:
Build file 'mockk/build.gradle.kts' line: 39

* What went wrong:
Script compilation errors:

  Line 39:     kover(projects.modules.mockkAndroid)
                                      ^ Unresolved reference: mockkAndroid

  Line 40:     kover(projects.modules.mockkAgentAndroid)
                                      ^ Unresolved reference: mockkAgentAndroid

```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -36,8 +36,13 @@ idea {
 dependencies {
     kover(projects.modules.mockk)
     kover(projects.modules.mockkAgent)
-    kover(projects.modules.mockkAndroid)
-    kover(projects.modules.mockkAgentAndroid)
+    // if the android SDK is not available trying to resolve the module names will fail
+    listOf("android", "agent-android").forEach { module ->
+        rootProject.subprojects.find { it.name == "mockk-$module" }?.let {
+            kover(it)
+        }
+    }
+
     kover(projects.testModules.loggerTests)
     kover(projects.testModules.clientTests)
     kover(projects.testModules.performanceTests)

